### PR TITLE
Completely remove cache-machine (bug 881405)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=881405

### CODE REVIEW

**Comment by chuckharmston** [comment]
tumblr_lk66co1hp91qb5cl9

File: mkt/site/models.py [hunk 1]
```diff
@@ -100,21 +84,14 @@ def __len__(self):
...
 
 
-class CachingRawQuerySet(RawQuerySet, caching.base.CachingRawQuerySet):
-    """A RawQuerySet with __len__ and caching."""
-
-
-# Make TransformQuerySet one of CachingQuerySet's parents so that we can do
-# transforms on objects and then get them cached.
-CachingQuerySet = caching.base.CachingQuerySet
-CachingQuerySet.__bases__ = (TransformQuerySet,) + CachingQuerySet.__bases__
```
Comment (robhudson) [inline mkt/site/models.py]: :eyes:

File: mkt/versions/models.py [hunk 2]
```diff
@@ -205,8 +205,8 @@ def transformer(cls, versions):
         if not versions:
             return
 
-        # FIXME: find out why we have no_cache() here and try to remove it.
-        files = File.objects.filter(version__in=ids).no_cache()
+        # FIXME: find out why we have here and try to remove it.
```
Comment (robhudson) [inline mkt/versions/models.py]: We can just remove this `FIXME` now

File: mkt/site/models.py (hunk 1 above)
Comment (diox) [inline mkt/site/models.py]: :eyeglasses:

**Comment by robhudson** [comment]
I'm really happy with this. Nice work. r+

**Pull request merged**